app/src/main/res/layout/fragment_calendar.xml

```diff
@@ -1,13 +1,19 @@
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
i'm a bit confused why you have this fragment layout twice?

Reply:
removed second layout.